freeipa/src/main/java/com/sequenceiq/freeipa/init/ModifyAdminPassordExpiration.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.freeipa.init;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.core.task.AsyncTaskExecutor;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.service.freeipa.FreeIpaClientFactory;
+import com.sequenceiq.freeipa.service.freeipa.user.UserService;
+import com.sequenceiq.freeipa.service.stack.StackService;
+
+@Component
+public class ModifyAdminPassordExpiration implements ApplicationListener<ContextRefreshedEvent> {
```

Comment:
Are we concerned about Spring triggering multiple ContextRefreshEvents?

Reply:
not really, code is idempotent, and we will drop it after it has been rolled out to prod